sdks/python/apache_beam/runners/interactive/dataproc/dataproc_cluster_manager.py

```diff
@@ -167,8 +166,7 @@ def create_flink_cluster(self) -> None:
             },
             'gce_cluster_config': {
                 'metadata': {
-                    'flink-start-yarn-session': 'true',
-                    'PIP_PACKAGES': 'apache-beam[gcp]=={}'.format(beam_version)
```

Comment:
So we sure that as long as we provide the environment_config, the installation of a Python SDK on the cluster node is not needed.